.clang-format

```
@@ -0,0 +1,27 @@
+BasedOnStyle: LLVM
```

Since this can be done for other languages, I suggest:
```suggestion
---
BasedOnStyle: LLVM
---
Language: Cpp
```

---

What is the additional benefit of the `Language: Cpp` here, since the GitHub statistics for this repo list that the celix repo contains 60% C code and 30% C++ code. 

(I don't know how accurate those stats are)

---

I saw that the project is using few JavaScript files (not many), so I thought to be explicit here that these are intended to C/C++.

---

As far as I know clang-format support Cpp as language, but not C. Also Celix is a mix of C and C++ (mainly C)

---

As far as I know, it can be used for C.